demos/common/cpp/models/src/detection_model_yolo.cpp

```diff
@@ -126,57 +134,73 @@ std::unique_ptr<ResultBase> ModelYolo3::postprocess(InferenceResult & infResult)
     return std::unique_ptr<ResultBase>(result);
 }
 
-void ModelYolo3::parseYOLOV3Output(const std::string& output_name,
+void ModelYolo::parseYOLOV3Output(const std::string& output_name,
```

Comment:
Rename to `parseYOLOOutput` or else without version.

Reply:
Done